plugins/catalog-import/config.d.ts

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2022 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export interface Config {
+  /**
+   * Configuration options for the catalog plugin.
+   */
+  catalog?: {
+    /**
+     * Catalog entity descriptor filename, defaults to "catalog-info.yaml"
+     * @visibility frontend
+     */
+    filename?: string;
```

Comment:
Hm. Thinking about the longer term and scope creep, it may be wise to make these have longer and more specific names. Perhaps `defaultEntityFilename` here? This is an OK config parameter to have, I think.

Reply:
Do you see this parameter being useful outside of catalog import? If not, should this be moved under `catalog.import.filename` for example?
Did a lot of back-and-forth with the name, how do you feel about `entityDescriptorFilename` or just `entityFilename`? I think having `default` in the name is obsolete as there's a default value for this parameter and if you end up changing it it's not default anymore.

Reply:
Settled for `entityFilename`